ajax/libs/hashgrid/package.json

```diff
@@ -4,6 +4,7 @@
   "version": "6",
   "description": "A little tool that inserts a layout grid in web pages, allows you to hold it in place, and toggle between displaying it in the foreground or background.",
   "homepage": "http://hashgrid.com/",
+  "license":　"Apache-2.0",
```

Comment:
@yangwanting 
There is a unvalid space.
Please modify it.
And remember write the correct library name in commit message next time.